mobify/progressive-web-scaffold
# Test docs and setup

A follow-up to the previous PR that adds boilerplate code for automated end-to-end tests. 

## Changes
- Short blurb in README
- Updates Chrome in the CircleCI container - Ubuntu 12.04 is running an old version of Chrome https://circleci.com/docs/build-image-precise/

## How to test-drive this PR
- View the README 
- See https://circleci.com/gh/mobify/crabtree-evelyn-progressive for example of updated Chrome on CircleCI
